apis/sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/BuiltInFunctionResource.java

```diff
@@ -124,8 +124,12 @@ public Uni<RestResponse<Object>> executeBuiltInFunction(
       @QueryParam("raw") boolean raw,
       @NotNull(message = "payload not provided") @Valid BuiltInFunctionDto body) {
     ExecutionContext context = ExecutionContext.create(profile);
+
     List<String> subPath =
-        documentPath.stream().map(PathSegment::getPath).collect(Collectors.toList());
+        documentPath.stream()
+            .map(PathSegment::getPath)
+            .filter(p -> !p.isEmpty())
```

Comment:
So empty path segments are allowed, just trimmed out?

Reply:
Yeah, if you do something like /api/v2/namespaces/x/collections/y/document/a//b/c, I just figured it should ignore the typo

Reply:
Yeaaaa that actually might be a problem in certain cases...it would mean that we have to restrict JSON fields from having empty string as well, or else you could write JSON with an empty string as field and then never be able to read that path directly via the URI. Let me check to see how we handle this, because also I think that empty-string serves as the delimiter for where the nesting ends in the cassandra row

Reply:
Right. To me it seems reasonable not to allow these, but I am not 100% sure how Ivan feels about that.
I could see empty String as key for some sort of special metadata, and it is probably used by someone somewhere.
Does not mean we have to support it but we need to decide one way or the other.